doc/source/conf.py

```diff
@@ -287,6 +291,7 @@
         "json_url": f"https://{cname}/versions.json",
         "version_match": get_version_match(__version__),
     },
+    "check_switcher": False,
```

Comment:
Why this one?

Reply:
As far as I know, the version switcher file is needed primarily for private repositories. In such cases, the theme cannot verify if the version switcher file is in the correct format. Additionally, it is necessary for local builds when providing a dummy CNAME for an environment variable. For instance, if you set cname = os.environ.get(DOC_CNAME, "noname.com"), the environment variable for the CNAME might not be present during a local build, causing the switcher to fail. However, if you provide the correct name, this requirement is unnecessary. especially for public libs

Reply:
Version switcher to False is only needed while the repo is private or if it is the first time you are publishing documentation on a public repo. Typically, it is the first case. Not needed here

Reply:
I'll remove it from PyAnsys Geometry - not need3ed anymore

Reply:
Thank you to both of you for the explanations! I will remove it